pages/[chainId]/[address]/index.tsx

```diff
@@ -125,24 +131,45 @@ export async function getStaticPaths() {
 export const getStaticProps = async (
   context: GetStaticPropsContext<QueryParams>,
 ) => {
+  let chainId = context.params?.chainId;
```

Comment:
should we rename the param `chainId` to `chainName` to be more clear ?

Reply:
what about just `network`?

Reply:
yeah sure. it's just that `chainId` in the context of this codebase is used for the hex string of the chain number and we just need another name here